contrib/macdeploy/gen-sdk

```diff
@@ -57,8 +57,8 @@ def run():
         out_sdktgz_path = pathlib.Path("./{}.tar.gz".format(out_name))
 
     def tarfp_add_with_base_change(tarfp, dir_to_add, alt_base_dir):
-        """Add all files in dir_to_add to tarfp, but prepent MEMBERPREFIX to the files'
-        names
+        """Add all files in dir_to_add to tarfp, but prepent MEMBERPREFIX to the file's
+        name
```

Comment:
I'm not sure this needs any fixing

Reply:
the original just read a little strange and just adding the apostrophe on its own as codespell suggested wouldn't remedy the issue. So this is more for easier readability